optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -25,11 +25,19 @@
 
 import numpy as np
 import torch
+from packaging.version import Version, parse
 from transformers import AutoConfig, AutoModelForSeq2SeqLM, AutoModelForSpeechSeq2Seq, AutoTokenizer
+from transformers import __version__ as transformers_version
 from transformers.file_utils import add_start_docstrings_to_model_forward, default_cache_path
 from transformers.generation_utils import GenerationMixin
 from transformers.modeling_outputs import BaseModelOutput, Seq2SeqLMOutput
 
+
+if parse(transformers_version) >= Version("4.25.0"):
```

Comment:
Same + could we rather have this this as `check_if_transformers_greater("x.x.x")` and the method in `utils/__init__.py`?

Reply:
Yes, that's a good suggestion. Since I already merged this PR, I'll do it in a new one.